src/rawcan_comm.cpp

```diff
@@ -42,7 +54,63 @@ namespace roboteq {
             printf("Error in socket bind");
             throw -2;
         }
+    }
+
+    bool rawcan_comm::download(runtime_command command, uint8_t subindex, uint32_t data) {
 
-    } 
+        struct can_frame frame;
+
+        if (rawcan_comm::runtime_command_map.find(command) == rawcan_comm::runtime_command_map.end()) {
+                printf("Error in command");
+            throw -1; 
+            }
+        
+        std::unordered_map<runtime_command, uint16_t>::const_iterator command_map = rawcan_comm::runtime_command_map.find(command);
+
+        frame.can_id  = 0x600 + roboteq::rawcan_comm::_roboteq_can_id;
+        frame.can_dlc = 8;
+        frame.data[0] = 0x20;
+        frame.data[1] = command_map->second;
+        frame.data[2] = command_map->second >> 8;
+        frame.data[3] = subindex;
+        frame.data[4] = data >> 0;
+        frame.data[5] = data >> 8;
+        frame.data[6] = data >> 16;
+        frame.data[7] = data >> 24;
+
+        write(roboteq::rawcan_comm::_socket_handle, &frame, sizeof(struct can_frame));
+
```

Comment:
when you write the sdo object, you always get a response acknowleding the receipt and stuff. We should probably read it and make sure its right.